pkg/patterns/addon/pkg/status/kstatus.go

```diff
@@ -88,7 +87,7 @@ func (k *kstatusAggregator) BuildStatus(ctx context.Context, info *declarative.S
 		}
 		currentStatus.Phase = string(aggregatedPhase)
 	}
-
+	currentStatus.Healthy = currentStatus.Phase == string(status.CurrentStatus)
```

Comment:
I'd like to confirm if the logic is correct, basically it means 
1. reconcile does not have errors like `InternalError`,  `VersionMismatch`, `FailedToApply` 
2. reconcile is not in progress, including `Terminating` and `InProgress`
3. reconcile is not failed.
5. all deployment manifests are reconciled, and no abnormal conditions are detected (this is guaranteed from the way how we calculate the `current` status)

If this logic is reasonable, the `status.healthy` and `status.phase` seems to be duplicate fields. I think it's fine since [`phase` is deprecated](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md), just want to double check this is what k-d-p wants.